src/Chapter2.hs

```diff
@@ -621,7 +629,9 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven (e:_:r) = e : takeEven r
+takeEven r       = r
```

Comment:
I would recommend here to cover the remaining cases explicitly to take advantage of GHC pattern matching checker coverage :+1: 
```suggestion
takeEven [x]       = [x]
takeEven []        = []
```